### DISCUSSION TITLE
Notations, Paradigms, and Little Languages (March 2024)

### DISCUSSION THREAD

**Discussion by ChrisShank** [Little-Languages/reading-club#10] [Meetups]
This meetup’s topic revolves around interconnectedness between notations, paradigms, and little languages.

## Materials
Materials can be found [here](https://github.com/Little-Languages/reading-club/tree/main/topics/notations%20and%20paradigms).

## Recording
The recording can be found [here](https://www.youtube.com/watch?v=F1sQHC05jMU).

**Comment by patrickgwsmith** [comment]
Recording: https://www.youtube.com/watch?v=F1sQHC05jMU

**Reply by ChrisShank** [reply]
Thanks, updated!
